packages/classification/src/components/classification-item.tsx

```diff
@@ -53,15 +53,15 @@ export function ClassificationItem({
   };
 
   const onClassificationChange = (event: FormEvent<{ value: string }>) => {
-    const classification = event.currentTarget?.value;
+    const classificationValue = event.currentTarget?.value;
     const onlyNumbersExpression: RegExp = /^\d*$/;
 
     //If the input contains any non-numeric character ignore
-    if (!classification.match(onlyNumbersExpression)) return;
+    if (!classificationValue.match(onlyNumbersExpression)) return;
 
     setValues({
-      ...values,
-      classification: parseInt(classification, 10),
+      name: classification.name,
+      classification: parseInt(classificationValue),
```

Comment:
any reason you removed the base?

Reply:
I saw that it uses decimal numeric system unless the string starts with 'x0' in which case it uses hexadecimal but wasn't aware that if the string starts with 0 it uses octal. So I'll change it to use the decimal explicitly again 😅